apps/site/components/Downloads/MinorReleasesTable/index.tsx

```diff
@@ -15,50 +15,69 @@ type MinorReleasesTableProps = {
   releases: Array<MinorVersion>;
 };
 
+const chunkedReleases = (releases: Array<MinorVersion>, size: number) => {
+  const count = Math.ceil(releases.length / size);
+
+  return Array.from({ length: count }, (_, index) =>
+    releases.slice(index * size, (index + 1) * size)
+  );
+};
```

Comment:
nit: This function isn't doing anything to the releases in particular, so we can make it a bit more generic, so if we need it in the future, it's not requiring the `MinorVersion` array:
```suggestion
const chunk = <T,>(array: Array<T>, size: number): Array<Array<T>> => {
  const count = Math.ceil(array.length / size);

  return Array.from({ length: count }, (_, index) =>
    array.slice(index * size, (index + 1) * size)
  );
};
```